packages/nimble-components/src/table-column/icon/tests/table-column-icon-matrix.stories.ts

```diff
@@ -0,0 +1,62 @@
+import type { StoryFn, Meta } from '@storybook/html';
+import { html, ViewTemplate } from '@microsoft/fast-element';
+import { createMatrixThemeStory } from '../../../utilities/tests/storybook';
+import { sharedMatrixParameters } from '../../../utilities/tests/matrix';
+import { Table, tableTag } from '../../../table';
+import { tableColumnIconTag } from '..';
+import { iconCheckTag } from '../../../icons/check';
+import { mappingIconTag } from '../../../mapping/icon';
+import { mappingSpinnerTag } from '../../../mapping/spinner';
+
+const metadata: Meta = {
+    title: 'Tests/Table Column Types',
+    parameters: {
+        ...sharedMatrixParameters()
+    }
+};
+
+export default metadata;
+
+const data = [
+    {
+        id: '0',
+        code: 0
+    },
+    {
+        id: '1',
+        code: 1
+    },
+    {
+        id: '2',
+        code: 2
+    }
+] as const;
```

Comment:
I think it would be good to expand this to include different icon severities. You could do this with a column for each severity or use the matrix test infrastructure to create a table for each severity.

In general the matrix tests should cover all the relevant attributes of the child mapping elements too. This is the only one I noticed but if you can think of others then let's consider including them too.

Reply:
Added. I think that's the only other attribute that makes sense to test visually.